diagnostic.md

```diff
@@ -54,7 +68,10 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+.big
+#contentPane
+.important .red
```

Comment:
`.important.red` is applying the styles to elements that have BOTH those classes on it. The space, act as a descendent relationship.